lib/home_page.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'utils/constants.dart';
+import 'utils/text_styles.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.amber,
+      body: SafeArea(
+          child: Center(
+        child: Column(
```

Comment:
Please review indentation. You can use option + command + L to indent all the code correctly

Reply:
done